tests/testthat/test-tables.R

```diff
@@ -258,3 +258,23 @@ test_that("a correct but slightly pathological table correctly parsed", {
                c(rep('Categorical Columns', 2),
                  rep('Numerical Columns', 2)))
 })
+
+test_that("Test Wiki table correctly parsed", {
+    table <- read_html("../testwiki.html") %>%
```

Comment:
Could you please replace these with the simplest possible inline data? Relying on an external website makes for fragile tests.